### PR TITLE
observing the world

### PR DESCRIPTION
**Pull request by psav**
Purpose or Intent
=================

let's see
{{pytest: cfme/tests/services/test_myservice.py --long-running}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 5dc321c68d9fb98b4cbc09636c140cf0a96beb58:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
You have multiple commits in the PR. When you want your PR to be merged, please squash them. Do not squash them when working on the pull request as this allows to see the progress!

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit b3e03f0c66756f1f1e2eaab3f14e88d432f465f7:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
You have multiple commits in the PR. When you want your PR to be merged, please squash them. Do not squash them when working on the pull request as this allows to see the progress!

*CFME QE Bot*

File: cfme/tests/services/test_myservice.py [hunk 1]
```diff
@@ -45,7 +46,9 @@ def myservice(setup_provider, provider, catalog_item, request):
     Metadata:
         test_flag: provision
     """
-    vm_name = catalog_item.provisioning_data["vm_name"] + "_0001"
+    vm_name = version.pick({
```
Comment (RonnyPfannschmidt) [inline cfme/tests/services/test_myservice.py]: would it make sense to standardize on minimizing the picked sub-expression? (i.e. just the vm prefix)

File: cfme/services/catalogs/catalog_item.py [hunk 2]
```diff
@@ -171,9 +170,6 @@ def create(self):
             tabstrip.select_tab("Catalog")
             template = template_select_form.template_table.find_row_by_cells({
                 'Name': self.catalog_name,
-                # HACK to workaround BZ1390209
```
Comment (mshriver) [inline cfme/services/catalogs/catalog_item.py]: duplicates changes from PR #3801

File: cfme/tests/services/test_myservice.py (hunk 1 above)
Comment (mshriver) [inline cfme/tests/services/test_myservice.py]: Duplicates changes made in PR #3801

Comment (psav) [inline cfme/tests/services/test_myservice.py]: We need to keep this as is. This is not us making up a name, this is a name that ManageIQ assigns to the VM.

**Comment by psav** [comment]
This PR was primarily to try out some additional fixes/updates to the auto

File: cfme/tests/services/test_myservice.py (hunk 1 above)
Comment (mshriver) [inline cfme/tests/services/test_myservice.py]: Thanks for the clarification.

**Comment by dajoRH** [comment]
Lint report for commit 9395d914bd0b9bdd8419c0d17ec907ac540935a4:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
You have multiple commits in the PR. When you want your PR to be merged, please squash them. Do not squash them when working on the pull request as this allows to see the progress!

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 3a93a26665c4384870af152e3ce34d41eb35ad2a:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
You have multiple commits in the PR. When you want your PR to be merged, please squash them. Do not squash them when working on the pull request as this allows to see the progress!

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 2c465b92f21eb5b93d1bebb954f46f3501b7ac03:

`conftest.py`:
- :bangbang: Line 34:1: **E302** *expected 2 blank lines, found 0*

Please, rectify these issues :smirk: .

Commit inspection report:

**No commit flaws detected**.
You have multiple commits in the PR. When you want your PR to be merged, please squash them. Do not squash them when working on the pull request as this allows to see the progress!

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit cc7e0d014a7a35eab55e5322a7e8c8f5af2d9b48:

`conftest.py`:
- :bangbang: Line 34:1: **E302** *expected 2 blank lines, found 0*

Please, rectify these issues :smirk: .

Commit inspection report:

**No commit flaws detected**.
You have multiple commits in the PR. When you want your PR to be merged, please squash them. Do not squash them when working on the pull request as this allows to see the progress!

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit c131d2b925f78349cea262e176564b015dc114b5:

`conftest.py`:
- :bangbang: Line 34:1: **E302** *expected 2 blank lines, found 0*

Please, rectify these issues :smirk: .

Commit inspection report:

**No commit flaws detected**.
You have multiple commits in the PR. When you want your PR to be merged, please squash them. Do not squash them when working on the pull request as this allows to see the progress!

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit ebf9331597cc619f4d57b230dcccb37e7a6bb7b4:

`conftest.py`:
- :bangbang: Line 34:1: **E302** *expected 2 blank lines, found 0*

Please, rectify these issues :smirk: .

Commit inspection report:

**No commit flaws detected**.
You have multiple commits in the PR. When you want your PR to be merged, please squash them. Do not squash them when working on the pull request as this allows to see the progress!

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 8ecda4ebdcdd32343aec7f21d7b5dd508150ae69:

`conftest.py`:
- :bangbang: Line 34:1: **E302** *expected 2 blank lines, found 0*

Please, rectify these issues :smirk: .

Commit inspection report:

**No commit flaws detected**.
You have multiple commits in the PR. When you want your PR to be merged, please squash them. Do not squash them when working on the pull request as this allows to see the progress!

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit f0cdc48f5ff654ac496ca6a0dfa3557987438e11:

`conftest.py`:
- :bangbang: Line 34:1: **E302** *expected 2 blank lines, found 0*

Please, rectify these issues :smirk: .

Commit inspection report:

**No commit flaws detected**.
You have multiple commits in the PR. When you want your PR to be merged, please squash them. Do not squash them when working on the pull request as this allows to see the progress!

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 15410b35471d2717db05f58932096ed6b901b149:

`conftest.py`:
- :bangbang: Line 34:1: **E302** *expected 2 blank lines, found 0*

Please, rectify these issues :smirk: .

Commit inspection report:

**No commit flaws detected**.
You have multiple commits in the PR. When you want your PR to be merged, please squash them. Do not squash them when working on the pull request as this allows to see the progress!

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit d4c5c658fdc821edbb975baba52e5a332b509ccd:

`conftest.py`:
- :bangbang: Line 34:1: **E302** *expected 2 blank lines, found 0*

Please, rectify these issues :smirk: .

Commit inspection report:

**No commit flaws detected**.
You have multiple commits in the PR. When you want your PR to be merged, please squash them. Do not squash them when working on the pull request as this allows to see the progress!

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 6a554122e534efef2945dc9fc855734027151c2b:

`conftest.py`:
- :bangbang: Line 34:1: **E302** *expected 2 blank lines, found 0*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit bd7b1c6a0bb50245e5f62ec1be46a25ca4a6724d:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit ed99ee18e37a05de9f50a504418324adcfac1850:

`cfme/services/myservice.py`:
- :red_circle: Line 10:1: **F401** *'current_appliance' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 1278ed1d20ff047e34ed4f98bbb64f21df19fca8:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by RonnyPfannschmidt** [comment]
@psav will this eventually merge or was there just to learn and it can close now?

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by psav** [comment]
Closing as most things handled already in another PR

**Pull request closed without merging**